Firestore/Example/Benchmarks/remote_document_cache_benchmark.mm

```diff
@@ -100,6 +113,40 @@ void Shutdown(FIRFirestore* db) {
   dispatch_semaphore_wait(done, DISPATCH_TIME_FOREVER);
 }
 
+void BM_QueryIndexFree(benchmark::State& state) {
+  int64_t matching_docs = state.range(0);
+  int64_t total_docs = state.range(1);
+
+  FIRFirestore* db = OpenFirestore();
+  auto collection = [db collectionWithPath:MakeNSString("docs-" + CreateAutoId())];
+  WriteDocs(collection, matching_docs, /* match= */ true);
```

Comment:
Nit: there should be no spaces in the comment indicating the parameter name (here and below).